shell/platform/darwin/ios/platform_message_handler_ios.mm

```diff
@@ -118,12 +118,16 @@ - (void)dispatch:(dispatch_block_t)block {
                                                   FlutterBinaryMessageHandler handler,
                                                   NSObject<FlutterTaskQueue>* task_queue) {
   FML_CHECK(platform_task_runner_->RunsTasksOnCurrentThread());
+  // Use `respondsToSelector` instead of `conformsToProtocol` to accomodate
+  // hacking in your own `FlutterTaskQueue`.
```

Comment:
can you explain more about "hacking in your own `FlutterTaskQueue`?

Reply:
done, `injecting` is a better description